src/oc_erchef/priv/reindex-opc-organization

```diff
@@ -158,7 +158,10 @@ init_network() ->
 find_dl_headers(OrgNameBin, IntLB) when is_binary(OrgNameBin) ->
     find_dl_headers(binary_to_list(OrgNameBin), IntLB);
 find_dl_headers(OrgName, IntLB) when is_list(OrgName) ->
-    {ok, "200", _Headers, Body} = rpc:call(?ERCHEF, ibrowse,send_req, [IntLB ++ "/_route/organizations/" ++ OrgName, [], get]),
+    Ssl1 = rpc:call(?ERCHEF, envy, get, [oc_chef_authz, authz_service, list]),
+    Ssl2 = proplists:get_value(ibrowse_options, Ssl1, []),
+    Ssl3 = proplists:delete(verify, proplists:get_value(ssl_options, Ssl2,[])),
```

Comment:
Let us add some context about why this is being done and the security considerations that we have made so that the erlang node/communications are not compromised.